src/GovUk.Education.ExploreEducationStatistics.Data.Model/Repository/BoundaryLevelRepository.cs

```diff
@@ -18,9 +19,32 @@ public BoundaryLevelRepository(StatisticsDbContext context)
             _context = context;
         }
 
+        public async Task<IEnumerable<BoundaryLevel>> Get()
+        {
+            return await _context.BoundaryLevel.ToListAsync();
+        }
+
         public Task<BoundaryLevel?> Get(long id)
         {
-            return _context.BoundaryLevel.SingleOrDefaultAsync(level => level.Id == id);
+            return id == 0
+                ? throw new ArgumentNullException(nameof(id))
+                : _context.BoundaryLevel.SingleOrDefaultAsync(level => level.Id == id);
+        }
+
+        public async Task Update(
+            long id,
+            string label)
+        {
+            if (id == 0)
+            {
+                throw new ArgumentNullException(nameof(id));
+            }
+
+            var level = await _context.BoundaryLevel.FindAsync(id)
+                ?? throw new KeyNotFoundException();
+
+            level.Label = label;
+            await _context.SaveChangesAsync();
         }
```

Comment:
See comment left on `Create` about adding an `Updated` datetime to audit when these get updated.